apps/explorer/src/components/orders/BridgeDetailsTable/index.tsx

```diff
@@ -0,0 +1,327 @@
+  const feeItems: ReactNode[] = []
+
+  if (bridgeDetails.gasCostsNative) {
+    feeItems.push(
+      <FeeItem key="gas-costs">
+        <FeeLabel>Native Gas Costs:</FeeLabel>
+        <FeeValue>
+          {bridgeDetails.gasCostsNative} on {getChainInfo(bridgeDetails.source.chainId as SupportedChainId).label}
+        </FeeValue>
+      </FeeItem>,
+    )
+  }
+
+  if (bridgeDetails.protocolFeeSellToken) {
+    feeItems.push(
+      <FeeItem key="protocol-fee-source">
+        <FeeLabel>Protocol Fee (Source):</FeeLabel>
+        <FeeValue>
+          {bridgeDetails.protocolFeeSellToken} {safeTokenName(bridgeDetails.source)}
+        </FeeValue>
+      </FeeItem>,
+    )
+  }
+
+  if (bridgeDetails.protocolFeeBuyToken) {
+    feeItems.push(
+      <FeeItem key="protocol-fee-dest">
+        <FeeLabel>Protocol Fee (Dest):</FeeLabel>
+        <FeeValue>
+          {bridgeDetails.protocolFeeBuyToken} {safeTokenName(bridgeDetails.destination)}
+        </FeeValue>
+      </FeeItem>,
+    )
+  }
+
+  if (feeItems.length > 0) {
+    return <FeesWrapper>{feeItems}</FeesWrapper>
+  }
+
+  return (
+    <FeeItem>
+      <FeeValue>Not available</FeeValue>
+    </FeeItem>
+  )
+}
+
+const tooltipTextMap = {
+  transactionHash: 'The transaction hash or provider-specific explorer link for the bridge operation.',
+  status: 'The current status of the bridge operation.',
+  amounts: 'The amount of tokens sent to the bridge and the amount received (or expected).',
+  costsAndFees: 'Estimated or actual costs and protocol fees for the bridge operation.',
+  bridgingTime: 'Expected time for the bridge operation to complete.',
+  maxSlippage: 'The maximum allowed slippage for the bridge in percentage.',
+  provider: 'The bridging solution provider.',
+  ownerAddress: 'The account address from which the tokens are bridged.',
+  receiverAddress: 'The account address to which the tokens are bridged on the destination chain.',
+}
+
+const mapBridgeableToErc20 = (
+  bridgeToken: BridgeDetails['source'] | BridgeDetails['destination'],
+): TokenErc20 & { chainId?: Network } => {
+  if (!bridgeToken) {
+    return { address: '', decimals: 18, symbol: 'ERR', name: 'Error Token' } as TokenErc20 & { chainId?: Network }
+  }
+  return {
+    address: bridgeToken.address,
+    decimals: bridgeToken.decimals === undefined ? 18 : bridgeToken.decimals,
+    symbol: bridgeToken.symbol || 'N/A',
+    name: bridgeToken.symbol || bridgeToken.address,
+    chainId: bridgeToken.chainId as Network,
+  }
+}
+
+export function BridgeDetailsTable({
+  bridgeDetails,
+  isLoading: isOverallLoading,
+  ownerAddress,
+  receiverAddress,
+}: {
+  bridgeDetails?: BridgeDetails
+  isLoading?: boolean
+  ownerAddress?: string
+  receiverAddress?: string
+}): ReactNode {
+  // Early return if no bridge details to render
+  if (!bridgeDetails && !isOverallLoading) {
+    return null
+  }
+
+  function getTransactionDetailsDisplay(): ReactNode {
```

Comment:
Still a render function.
https://dev.to/adevnadia/react-re-renders-guide-preventing-unnecessary-re-renders-21dm?comments_sort=oldest
⛔️ Antipattern: Creating components in render function